web/styles/settings.css

```diff
@@ -396,8 +396,12 @@ td .button {
     }
 }
 
-.user_role {
-    min-width: 95px;
+.user_id {
+    min-width: 60px !important;
```

Comment:
We generally avoid using `!important`. Would be better to instead use a more specific selector.